internal/app/validate.go

```diff
@@ -155,11 +156,17 @@ func checkImage(pod *v1.Pod,
 				"image": image,
 			})
 		} else {
-			message = fmt.Sprintf("Image passed policy check: %s", image)
-
-			logger.Warn("Image passed policy check", map[string]interface{}{
-				"image": image,
-			})
+			if isWl {
+				message = fmt.Sprintf("Whitelisted release: %s", image)
+				logger.Warn("Whitelistetd release", map[string]interface{}{
```

Comment:
wouldn't use warn level here. this is normal operation, no need for extra attention from operator
```suggestion
				logger.Info("skipping policy enforcement for whitelisted release", map[string]interface{}{
```